_rules/attr-not-duplicated-e6952f.md

```diff
@@ -10,6 +10,11 @@ accessibility_requirements:
     failed: not satisfied
     passed: further testing needed
     inapplicable: further testing needed
+  wcag-technique:H94: # Ensuring that elements do not contain duplicate attributes
+    forConformance: true
```

Comment:
Techniques are not required for conformance.

Reply:
See https://github.com/act-rules/act-rules.github.io/pull/1014#discussion_r346227590
I'm resolving this conversation to keep the discussion in one place.